GTG/core/clipboard.py

```diff
@@ -56,7 +56,6 @@ def copy(self, start, stop, bullet=None):
                 if hasattr(ta, 'is_subtask'):
                     is_subtask = True
                     tid = ta.child
-                    tas = self.ds.tasks.lookup[tid]
```

Comment:
I wasn't sure what the clipboard is supposed to do, so I didn't check whether there is a bug here.

Copy/pasting the contents of a task with subtasks doesn't show the subtasks as subtasks, just text. But that's also the behaviour in 0.6.

Reply:
Yeah, the clipboard is supposed to handle GTG content. That's probably an old bug